libsolidity/interface/CompilerStack.cpp

```diff
@@ -971,6 +989,36 @@ void CompilerStack::generateIR(ContractDefinition const& _contract)
 	tie(compiledContract.yulIR, compiledContract.yulIROptimized) = generator.run(_contract);
 }
 
+void CompilerStack::generateEWasm(ContractDefinition const& _contract)
+{
+	solAssert(m_stackState >= AnalysisSuccessful, "");
+	Contract& compiledContract = m_contracts.at(_contract.fullyQualifiedName());
+	solAssert(!compiledContract.yulIROptimized.empty(), "");
+	if (!compiledContract.eWasm.empty())
+		return;
+
+	// Re-parse the Yul IR in EVM dialect
```

Comment:
Why is this needed?

Reply:
Because we only have the text representation at this stage.